### ISSUE TITLE
Replace TextKeyNode subclasses with static methods

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/scenery-phet#631]
Noted while converting to ES6 `class` for https://github.com/phetsims/tasks/issues/1044.

scenery-phet/js/keyboard/ contains a bunch of subclasses of `TextKeyNode` that all look like this:

```js
const keyEscString = sceneryPhetStrings.key.esc;

class EscapeKeyNode extends TextKeyNode {
  /**
   * @param {Object} [options]
   */
  constructor( options ) {
    super( keyEscString, options );
  }
}

sceneryPhet.register( 'EscapeKeyNode', EscapeKeyNode );
export default EscapeKeyNode;
```

Rather than having a bunch of classes that don't do anything except provide a specific string (and have to convert them to ES6 class!), it would be nicer to have static methods on TextKeyNode.  For example, the above class can be replaced with:

```js
@static escape( options ) {
  return new TextKeyNode( sceneryPhetStrings.key.esc, options );
}
```

I discussed with @jessegreenberg (the author) and he agreed that this would be a good change.

**Comment by pixelzoom** [comment]
Done in the above commits. After consulting with @jessegreenberg, we decided to use the same names for string keys and TextKeyNode method names. For example, `sceneryPhetStrings.key.alt` is rendered by `TextKeyNode.alt()`.

I reviewed the keyboard help in BASE, coulombs-law, GFL, molarity, ratio-and-proportion, and scenery-phet demo. I didn't see any problem, but it would be good to have a second look.

@jessegreenberg over to you for review.
